src/components/Contacts/ContactDetails/ContactDonationsTab/PartnershipInfo/EditPartnershipInfoModal/EditPartnershipInfoModal.test.tsx

```diff
@@ -538,4 +556,52 @@ describe('EditPartnershipInfoModal', () => {
       }),
     );
   });
+
+  describe('Relationship code', () => {
+    it('should not render relationshipCode', async () => {
+      const { queryByRole } = render(<Components />);
+
+      await waitFor(() => {
+        expect(mutationSpy).toHaveGraphqlOperation('UserOrganizationAccounts');
+      });
+
+      expect(
+        queryByRole('textbox', {
+          name: 'Relationship Code',
+        }),
+      ).not.toBeInTheDocument();
+    });
+
+    it('should render relationshipCode', async () => {
+      const { findByRole } = render(<Components includeCruSwitzerland />);
+
+      expect(
+        await findByRole('textbox', {
+          name: 'Relationship Code',
+        }),
+      ).toBeInTheDocument();
+    });
+
+    // Uncomment this when the UpdateContact mutation operation is updated to include the relationshipCode
+    // it('should update relationshipCode', async () => {
```

Comment:
We could use `it.skip` instead of commenting out the test. That way the body at least gets type checked, formatted, and code highlighted.

```suggestion
    it.skip('should update relationshipCode', async () => {
```